baccmod/base_acceptance_map_creator.py

```diff
@@ -864,6 +868,41 @@ def _get_interpolated_background(self, interp_func: interp1d, zenith: u.Quantity
 
         return interp_bkg
 
+    def _interpolate_bkg_to_energy_axis(self, data_bkg: u.Quantity, energy_axis_computation: MapAxis):
+        """
+            Compute the final background model from the provided data by interpolating the energy axis used for computation to the ones for the model
+
+            Parameters
+            ----------
+            data_bkg : u.Quantity
+                The data cube of the background model. The energy axis need to be the first one.
+            energy_axis_computation : gammapy.maps.geom.MapAxis
```

Comment:
Why pass this as argument instead of from self, if the energy_axis is accessed through self?

Reply:
It's in preparation of the next PR with a dynamic per node energy axis (with grouped binning)

Reply:
ok